pupil-spa/src/app/submission-pending/submission-pending.component.scss

```diff
@@ -1,16 +1,18 @@
-.spinner {
+img.spinner {
```

Comment:
how is the spinner being loaded?

Reply:
this didn't work with css so I had to add it in html instead as base64.

Reply:
It did work in disconnected mode as well.